src/main/scala/zio/kafka/admin/AdminClient.scala

```diff
@@ -891,13 +891,24 @@ object AdminClient {
     def apply(ri: JReplicaInfo): ReplicaInfo = ReplicaInfo(ri.size(), ri.offsetLag(), ri.isFuture)
   }
 
-  def make(settings: AdminClientSettings): ZManaged[Has[Blocking.Service], Throwable, AdminClient] =
-    ZManaged.service[Blocking.Service].flatMap { blocking =>
-      ZManaged.make(
-        ZIO(JAdminClient.create(settings.driverSettings.asJava)).map(ac => LiveAdminClient(ac, blocking))
-      )(client => ZIO.effectTotal(client.adminClient.close(settings.closeTimeout)))
+  def make(settings: AdminClientSettings): ZManaged[Blocking, Throwable, AdminClient] =
+    fromManagedJavaClient(javaClientFromSettings(settings))
+
+  def fromJavaClient(javaClient: JAdminClient): URIO[Blocking, AdminClient] =
+    ZIO.service[Blocking.Service].map { blocking =>
+      new LiveAdminClient(javaClient, blocking)
+    }
+
+  def fromManagedJavaClient[R, E](
+    managedJavaClient: ZManaged[R, E, JAdminClient]
+  ): ZManaged[Blocking & R, E, AdminClient] =
+    managedJavaClient.flatMap { javaClient =>
+      ZManaged.fromEffect(fromJavaClient(javaClient))
     }
 
+  def javaClientFromSettings(settings: AdminClientSettings): ZManaged[Any, Throwable, JAdminClient] =
```

Comment:
private maybe?

Reply:
I will actually use that one in devtools so no.